### PR TITLE
Partial translation improvement

### PR DESCRIPTION
**Pull request by DMSalesman**
I updated the Italian translation, though some terms are still unclear to me, hence I kept them untranslated.

Since I work from mobile, and GitHub is not easily usable by this means, could some maintainer mail me both the English and Italian "strings.xml" files?

### CODE REVIEW

**Comment by kuc** [comment]
@DMSalesman 
https://raw.githubusercontent.com/anthonycr/Lightning-Browser/dev/app/src/main/res/values/strings.xml

https://raw.githubusercontent.com/anthonycr/Lightning-Browser/dev/app/src/main/res/values-it/strings.xml

**Comment by DMSalesman** [comment]
Excellent, thanks.

Another thing: as I previously said, some of the settings are a bit obscure, and there are just a few things that I dislike more than an inaccurate translation.

Is it okay for me to pester any of you in this issue, should I need some clarification concerning a certain feature's usage?

**Comment by kuc** [comment]
>Is it okay for me to pester any of you in this issue, should I need some clarification concerning a certain feature's usage?

I see no problem in this, e-mail notifications don't require immediately response from me...

**Comment by DMSalesman** [comment]
And you may take all the time you need, either.

To begin, I would like to know what purpose does the option "Use wide viewport" (wideViewPort) have. I tried to find out by testing, but was unable to ascertain it.

Il 03 novembre 2016 15:52:55 CET, "Miłosz" <notifications@github.com> ha scritto:

**Comment by DMSalesman** [comment]
Please disregard the quoting of your previous message. I sometimes forget to delete it.

**Comment by kuc** [comment]
@DMSalesman See here:
https://developer.android.com/reference/android/webkit/WebSettings.html#setUseWideViewPort(boolean)

>Sets whether the WebView should enable support for the "viewport" HTML meta tag or should use a wide viewport. When the value of the setting is false, the layout width is always set to the width of the WebView control in device-independent (CSS) pixels. When the value is true and the page contains the viewport meta tag, the value of the width specified in the tag is used. If the page does not contain the tag or does not provide a width, then a wide viewport will be used.


Sooo, search for similar option in e.g. mobile Firefox and see what they translated it to your language, aiming for non-technical users 🙂 Something like _"Fit mobile pages to screen (width)"_ or other.

**Comment by DMSalesman** [comment]
Perfect. I experimented a bit, and I think I can come up with an understandable definition.

**Comment by DMSalesman** [comment]
Is the string identified by "problem_location_download" supposed to appear in a toast or in a pop-up?

**Comment by kuc** [comment]
Just search in repo code:
https://github.com/anthonycr/Lightning-Browser/blob/b2c6b3df732dc878fb02bfb60924d75dfea6ef95/app/src/main/java/acr/browser/lightning/download/DownloadHandler.java#L220

tl;dr: in snackbar (modern toast)

**Comment by DMSalesman** [comment]
Many thanks. I would search by myself if I were as proficient in Java as I am in Python. Sorry for the nuisance.

Second question: does the string "download_sdcard_busy_dlg_msg" concern a scenario in which the phone's storage is being accessed from a computer?

**Comment by kuc** [comment]
>Environment.MEDIA_SHARED
>Storage state if the media is present not mounted, and shared via USB mass storage.

So, YES 🙂

**Comment by DMSalesman** [comment]
Rest assured for my questions are coming to an end. Slowly.

Next, what about "color_mode"? What does it do?

Again, we're almost done, so keep yourself cool and focus, pal.

**Comment by DMSalesman** [comment]
Never mind, figured it out.

**Comment by kuc** [comment]
🆗

**Comment by DMSalesman** [comment]
Last questions, man.

What's the so-called "Web Storage", mentioned for example in string "clear_web_storage"? What does Web Storage do?

**Comment by kuc** [comment]
@DMSalesman https://en.wikipedia.org/wiki/Web_storage

**Comment by DMSalesman** [comment]
@kuc
That was the last one. I'll then close this pull request and open a new one, with the finalized translation.

Even if you're a Java and possibly other languages dev, you'd better keep in mind that you won't be able to present others with an acceptable user experience, without translators to localize your software for you. Arrogance is not a way to encourage contributors, gentleman.

**Comment by kuc** [comment]
>Even if you're a Java and possibly other languages dev, you'd better keep in mind that you won't be able to present others with an acceptable user experience, without translators to localize your software for you. Arrogance is not a way to encourage contributors, gentleman.

?
